lib/honeybadger/plugins/autotuner.rb

```diff
@@ -0,0 +1,30 @@
+require 'honeybadger/instrumentation_helper'
+require 'honeybadger/plugin'
+
+module Honeybadger
+  module Plugins
+    module Autotuner
+      Plugin.register :autotuner do
+        requirement { config.load_plugin_insights?(:autotuner) && defined?(::Autotuner) }
+
+        execution do
+          singleton_class.include(Honeybadger::InstrumentationHelper)
+
+          ::Autotuner.enabled = true
+
+          ::Autotuner.reporter = proc do |report|
+            Honeybadger.event("report.autotuner", report: report.to_s)
+          end
+
+          metric_source 'autotuner'
+
+          ::Autotuner.metrics_reporter = proc do |metrics|
```

Comment:
I think it's something to consider, however, the `autotuner` [documentation recommends](https://github.com/Shopify/autotuner/blob/main/README.md?plain=1#L35) to use something that would calculate averages or percentages. Part of it is because the frequency of the reporting is done by the `autotuner` gem which I don't see any configuration for. Eventing for these metrics might end up using more data than needed.

Reply:
Cool, thanks!